docs/api-docs/storefront/graphql/graphql-api-overview.mdx

```diff
@@ -708,6 +710,10 @@ If you wish to use the GraphQL Storefront API from an external system, there are
 
 As a best practice, you should create tokens that expire and rotate them regularly before their expiry. However, you are also permitted to create long-lived tokens.
 
+### Can I hide the display of a field, such as the product's price, on the storefront by toggling the setting in the control panel?
+
+You can toggle off the field `show_product_price` in the control panel to prevent the display in GraphQL.
```

Comment:
```suggestion
You can toggle off the field `show_product_price` in the control panel to prevent the display in GraphQL. It will make `prices` field to return `null` in the GQL response.
```

Reply:
Ahh yeah I like this more 👍  It's good to clarify that it'll return `null` to be more explicit.